Automattic/pocket-casts-android
# Display Episode Artwork In Miniplayer, Notification & Widget

## Description
In #1599 , we added episode artwork to the full-screen player and the episode details screen, and we now want to use it in more places in the app. See #1608 for more details. I've added artwork in MiniPlayer, Notification and Widget. 

## Testing Instructions

**Testing the Miniplayer, Notification & Widget**
1. Fresh install the app
2. Open up the podcast screen for "Darknet Diaries"
3. Open the episode details page for an episode and see the artwork of episode.
4. When network connection active or inactive, observe the Miniplayer, Notification & Widget artwork loaded from episode if successfully loads, If fails, automatically load the artwork of podcast.

## Screenshots or Screencast 
![00ecb5ad-3f03-4283-b175-20c70ed7b9c9](https://github.com/Automattic/pocket-casts-android/assets/64830777/8715cce7-ab9d-4157-a710-b4cd62f9c61b)
![83676f7d-99ca-42dd-b3f3-63b34d745a74](https://github.com/Automattic/pocket-casts-android/assets/64830777/d55e6534-133f-4cf5-9653-eb408db27bc8)
![a1d5d5f9-7004-4431-aa94-f165092a0ff5](https://github.com/Automattic/pocket-casts-android/assets/64830777/c06c798d-7b4b-470f-b334-82a570dee911)

## Checklist
 
#### I have tested any UI changes...
